SU-HSDO/suhumsci
# SHS-5619: CSS Code Cleanup: Icon mixins cleanup and improvements

# READY FOR REVIEW

## Summary
Icon mixins cleanup and improvements

## Need Review By (Date)
06/14

## Urgency
medium

## Steps to Test
1. On Colorful and Traditional, visit pages that have patterns carousels and cards with icons
2. Confirm the icons still look good

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
